Simplenote/Controllers/WidgetController.swift

```diff
@@ -0,0 +1,31 @@
+import WidgetKit
+
+struct WidgetController {
+    @available(iOS 14.0, *)
+    static func resetWidgetTimelines() {
+        WidgetCenter.shared.getCurrentConfigurations { result in
+            guard case .success(let widgets) = result else {
+                return
+            }
+
+            if widgets.contains(where: { (widget) -> Bool in
+                switch widget.configuration {
+                case is NoteWidgetIntent:
+                    return true
+                case is ListWidgetIntent:
+                    return true
+                default:
+                    return false
+                }
+            }) {
+                WidgetCenter.shared.reloadAllTimelines()
+            }
+        }
+    }
+
+    static func syncWidgetDefaults(authenticated: Bool) {
+        let widgetDefaults = WidgetDefaults.shared
+        widgetDefaults.sortMode = Options.shared.listSortMode
+        widgetDefaults.loggedIn = authenticated
+    }
```

Comment:
It was done this way to keep the WidgetsController from having any knowledge of simperium (which is where that authenticated value comes from)  It could be made to pass in both values, so the widget controller only sync what it is told at sync time